### ISSUE TITLE
Malaware warning in NPM audit

### ISSUE THREAD

**Issue by dlippy** [BlueWallet/react-native-blue-crypto#5]
Just FYI someone is trying to upload a modified version of this package to NPM that is compromised.  NPM audit returns:

react-native-blue-crypto  *
Severity: critical
Malware in react-native-blue-crypto - https://github.com/advisories/GHSA-9gh9-3fw4-pv6j
No fix available
node_modules/react-native-blue-crypto

**Comment by Overtorment** [comment]
yes, im aware, we dont use npm, we use this package directly from github

**Comment by astriskit** [comment]
Hoping @Overtorment meant that this is not dangerous anymore.

**Comment by Overtorment** [comment]
> Hoping [@Overtorment](https://github.com/Overtorment) meant that this is not dangerous anymore.

it was never dangerous if you use this package from github, _not_ from npm. i dont publish my packages to npm
